tools/src/tensil/tools/compiler/lir/Broadcast.scala

```diff
@@ -0,0 +1,97 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright © 2019-2022 Tensil AI Company */
+
+package tensil.tools.compiler.lir
+
+import tensil.tools.compiler.{LIR, InstructionContext, MemoryAddress, MemoryAddressRaw}
+
+class Broadcast(lirs: LIR*) extends LIR {
```

Comment:
Yes! I considered doing this, but it would make the changes even bigger and harder to review. I want to do a follow-up PR with this refactoring for both LIR and HIR. Wdyt?

Reply:
Added https://app.shortcut.com/tensil/story/460/refactor-hir-and-lir-to-class-per-op-design